.editorconfig

```diff
@@ -0,0 +1,14 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
```

Comment:
Won't this break Github-style markdown where you indicate a `<br>` via two spaces at the end of line?

Reply:
oh my. _i did not know that was a thing_. I think github preserves newlines in markdown.
without
spaces
at
the
end
of
the
line
;-)

Reply:
Yeah, Github uses simplified end line rules in comments :upside_down_face: 
But in `README.md` you need to put blank line for a new paragraph, otherwise it will be rendered as a single line.

Reply:
oh whut. every day a school day when nothing is consistent.

I copied this config from js-ipfs, so that's an interesting find @lidel

Reply:
@lidel i've added an override for markdown files

```
[*.md]
trim_trailing_whitespace = false
```